src/libraries/Microsoft.Extensions.Logging.Console/src/AnsiParser.cs

```diff
@@ -59,11 +59,11 @@ public void Parse(string message)
                 {
                     if (span[i + 3] == 'm')
                     {
-    #if (NETSTANDARD2_0 || NETFRAMEWORK)
-                        if (ushort.TryParse(span.Slice(i + 2, length: 1).ToString(), out ushort escapeCode))
-    #else
+#if NETCOREAPP
                         if (ushort.TryParse(span.Slice(i + 2, length: 1), out ushort escapeCode))
```

Comment:
I am still thinking it is worth it to parse manually and avoid the whole #if usage.